src/components/Wizard/CreateVmWizard/tests/CreateVmWizard.test.js

```diff
@@ -1,451 +1,88 @@
-  expect(component.find(NetworksTab)).toHaveLength(1);
-  expect(getNextButton(component).text()).toEqual(NEXT);
-
-  getNextButton(component).simulate('click');
-  component.update();
-
-  expect(component.state().activeStepIndex).toEqual(
-    getStepIndex(component.instance().wizardStepsNewVM, STORAGE_TAB_KEY)
-  );
-  expect(component.find(Loading)).toHaveLength(1);
-  expect(component.find(StorageTab)).toHaveLength(0);
-
-  const persistentVolumeClaims = [...CreateVmWizardFixutre[0].props.persistentVolumeClaims];
-  const storageClasses = [...CreateVmWizardFixutre[0].props.storageClasses];
-  component.setProps({ persistentVolumeClaims, storageClasses });
-
-  expect(component.find(Loading)).toHaveLength(0);
-  expect(component.find(StorageTab)).toHaveLength(1);
-
-  expect(component.state().activeStepIndex).toEqual(
-    getStepIndex(component.instance().wizardStepsNewVM, STORAGE_TAB_KEY)
-  );
-  expect(getNextButton(component).text()).toBe(createText);
-  expect(component.instance().lastStepReached()).toBeFalsy();
-
-  component.instance().onStepDataChanged(STORAGE_TAB_KEY, [{}], false); // create empty disk
-  component.update();
-
-  expect(getBackButton(component).props('disabled')).toBeTruthy();
-  component.instance().onStepDataChanged(
-    STORAGE_TAB_KEY,
-    [
-      {
-        id: 1,
-        name: 'D',
-        size: '15',
-        storageClass: 'iscsi',
-        isBootable: true,
-      },
-      {
-        id: 2,
-        isBootable: false,
-        attachStorage: {
-          id: 'disk-two',
-          name: 'disk Two',
-          size: '15',
-          storageClass: 'glusterfs',
-        },
-      },
-    ],
-    true
-  );
 
-  expect(createFunc).not.toHaveBeenCalled();
-  getNextButton(component).simulate('click');
-  component.update();
-
-  expect(component.find(ResultTab)).toHaveLength(1);
-  expect(component.state().activeStepIndex).toEqual(
-    getStepIndex(component.instance().wizardStepsNewVM, RESULT_TAB_KEY)
+const testCreateVmWizard = createTemplate => {
+  const [loaded] = CreateVmWizardFixutre;
+  const { props } = loaded;
+  return (
+    <Provider store={store}>
+      <CreateWizard {...props} createTemplate={createTemplate} />
+    </Provider>
   );
-  expect(component.instance().lastStepReached()).toBeTruthy();
-  expect(createFunc).toHaveBeenCalled();
-  expect(getBackButton(component).props('disabled')).toBeTruthy();
-  getBackButton(component).simulate('click'); // should not allow going backwards
-  expect(component.find(ResultTab)).toHaveLength(1);
-  expect(component.find(StorageTab)).toHaveLength(0);
 };
 
 describe('<CreateVmWizard />', () => {
   beforeEach(() => {
-    createVm.mockClear();
+    console.warn = jest.fn(); // redux Id warning when not properly connected to the store
+  });
+
+  it('it renders', () => {
+    const component = mount(testCreateVmWizard());
+    expect(component.html()).toMatchSnapshot(); // expect(component).toMatchSnapshot() freezes
```

Comment:
I could not make this work - jest freezes here (maybe it has something to do with redux?). So I at least used html to create the snapshot